azure-eventhubs-eph/src/test/java/com/microsoft/azure/eventprocessorhost/SmokeTest.java

```diff
@@ -33,6 +32,19 @@ public void SendRecv1MsgTest() throws Exception
 
 		testFinish(settings, SmokeTest.ANY_NONZERO_COUNT);
 	}
+        
+        @Test
+	public void ReceiverRuntimeMetricsTest() throws Exception
+	{
+		PerTestSettings settings = new PerTestSettings("ReceiverRuntimeMetrics");
+                settings.inOptions.setReceiverRuntimeMetricEnabled(true);
+		settings = testSetup(settings); 
+
+		settings.outUtils.sendToAny(settings.outTelltale);
+		waitForTelltale(settings);
+
+		Assert.assertTrue(settings.outProcessorFactory.getOnEventsContext().getRuntimeInformation() != null);
+	}
```

Comment:
Add "testFinish(settings, SmokeTest.ANY_NONZERO_COUNT);" at the end of this case. Otherwise, a run of all cases in this class will fail because this case will interfere with following ones.